src/analyz/mod.rs

```diff
@@ -61,11 +65,12 @@ impl<'tcx> ToJson<'tcx> for mir::AggregateKind<'tcx> {
             &mir::AggregateKind::Adt(_, _, _, _, _) => {
                 panic!("adt should be handled upstream")
             }
-            &mir::AggregateKind::Closure(ref defid, ref closuresubsts) => {
+            &mir::AggregateKind::Closure(ref defid, ref substs) => {
                 json!({
                     "kind": "Closure",
                     "defid": defid.to_json(mir),
-                    "closuresubsts": closuresubsts.substs.to_json(mir)
+                    // FIXME rename
```

Comment:
Is this left unchanged because it'll require changes to the `mir-verifier` parser, too?